command/ca/provisionerbeta/add.go

```diff
@@ -56,6 +57,11 @@ func addCommand() cli.Command {
 [**--admin-subject**=<subject>] [**--password-file**=<file>] [**--ca-url**=<uri>]
 [**--root**=<file>] [**--context**=<name>]
 
+**step beta ca provisioner add** <name> **--type**=NEBULA **--nebula-root**=<file>
```

Comment:
Maybe the type should be `Nebula`, like the one for Azure. Other types are abbreviations of systems. Nebula is a concrete system.

Reply:
Sure I didn't notice we were using different letter cases.

Reply:
Fixed with bf0a16f